packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount.tsx

```diff
@@ -168,13 +172,14 @@ const Amount = observer(({ is_minimized, is_nativepicker }) => {
                 duration_unit={duration_unit}
                 expiry_type={expiry_type}
                 onChange={onChange}
-                value={parseInt(is_equal)}
+                value={Number(is_equal)}
```

Comment:
@henry-deriv please elaborate on what error/warning is the reason to replace parseInt( with Number( here🙏🏻

Reply:
Argument of type 'number' is not assignable to parameter of type 'string'.ts(2345)

function parseInt(string: string, radix?: number | undefined): number

parseInt expects a `string` by typeof `is_equal` is `number`